pkg/project/resource_project_repo.go

```diff
@@ -127,10 +128,23 @@ var updateRepos = func(projectKey string, terraformRepoKeys []RepoKey, m interfa
 	return readRepos(projectKey, m)
 }
 
+var retryableOnUnresponsiveService = func(response *resty.Response, err error) bool {
+
+	retryableHttpStatusCode := []int{
```

Comment:
Using a switch statement is exactly the same number of linea of code but without creating a whole extra function.

I said this in my last comment. I am not a big fan of case statements, but that's the best solution go offers